src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -40,6 +40,7 @@ import AppInfoItem from './AppInfoItem'
 import DeployModeModal from './DeployModeModal'
 import { useLanguageContext } from '../../../providers/LanguageProvider'
 import { EXPLORER_BASE_URL } from '../../../utils/config'
+import { currencyType as defaultCurrencyType } from '../../../utils/i18n'
```

Comment:
This should already be defaulting to USD in the accountSlice.ts if RNLocalize.getCurrencies() returned undefined.

In order to encounter this bug, what is being returned by RNLocalize.getCurrencies(). I think the actual fix will be to check if the currency returned by RNLocalize.getCurrencies() is one of the supported currencies and if not we need to default to USD. We can only support currencies that are listed on coingecko via its API https://api.coingecko.com/api/v3/coins/helium?localization=false&tickers=false&community_data=false&developer_data=false

Reply:
I found this problem because 'RNLocalize.getCurrencies()' returned 'CNY'. We can set all unsupported currencies to USD, but if coingecko adds a new fiat currency, and we can only support it by updating the app, that's a bit of a pain.

Reply:
I just checked, although we support Japanese and Korean, we also do not support `JPY` and `KRW`. Coingecko supports both. Also users in both places can't select USD, because they can't open the currency selector.

Reply:
So we are missing a check to ensure that the users currency is not in the supported currencies list, and we can update the supported currencies to include a few more if they are listed in the coingecko api

Reply:
Yes, I look forward to a more reasonable solution from you. I'm closing this merge for now. Thanks a lot!